### PR TITLE
[D1] Add `wrangler-proxy` to community-projects.md

### PR DESCRIPTION
**Pull request by chientrm**
### wrangler-proxy

`wrangler-proxy` exposes Workers API (including D1) to the outside world, so you can integrate it with your favorite frameworks. It is compatible with DrizzleORM. It was originally created to enable Workers API in local development environments, but can also be used to self-host Workers runtime.

* [wrangler-proxy](https://github.com/chientrm/wrangler-proxy)

### CODE REVIEW

File: content/d1/platform/community-projects.md [hunk 1]
```diff
@@ -67,6 +67,12 @@ Instead of running the `wrangler d1 execute` command in your terminal every time
 
 * [GitHub](https://github.com/isaac-mcfadyen/d1-console)
 
+### wrangler-proxy
+
+`wrangler-proxy` exposes Workers API (including D1) to the outside world, so you can integrate it with your favorite frameworks. It is compatible with DrizzleORM. It was originally created to enable Workers API in local development environments, but can also be used to self-host Workers runtime.
```
Comment (Maddy-Cloudflare) [inline content/d1/platform/community-projects.md]:
```suggestion
`wrangler-proxy` exposes Workers API (including D1) to the outside world, so you can integrate it with your favorite frameworks. It is compatible with DrizzleORM. `wrangler-proxy` was originally created to enable Workers API in local development environments, but can also be used to self-host Workers runtime.
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Comment by admah** [comment]
@chientrm can you elaborate on what this provides that makes it different than `wrangler dev` or `wrangler pages dev`?

**Comment by chientrm** [comment]
> @chientrm can you elaborate on what this provides that makes it different than `wrangler dev` or `wrangler pages dev`?

You can use it as an add-on to other frameworks like SvelteKit, NextJS, ... It exposes the Workers API to the world and provides a wrapper API client .

**Comment by KimJ15** [comment]
👋Hi there, we’re going to close this PR due to inactivity

**Pull request closed without merging**